resources/lib/playbackmanager.py

```diff
@@ -37,7 +38,7 @@ def launch_up_next(self):
                 self.log('Error: no episode could be found to play next...exiting', 1)
                 return
         self.log('episode details %s' % episode, 2)
-        self.clock_twelve = 'm' in xbmc.getInfoLabel('System.Time').lower()
+        self.clock_twelve = bool(not get_global_setting('locale.use24hourclock', True))
```

Comment:
I'm not sure about the second parameter for `get_global_setting()` since I see below in [utils.py](https://github.com/im85288/service.upnext/blob/16625400ec961add172428d6fabde91373bc064d/resources/lib/utils.py#L169)
```
169 def get_global_setting(setting):
```

Reply:
Correct, our implementation of get_global_setting doesn't accept a default value. No worries, the new implementation gets rid of everything :-)